test/functional/transactions_tests.js

```diff
@@ -71,10 +76,10 @@ describe('Transactions', function() {
 
   describe('startTransaction', function() {
     it('should error if transactions are not supported', {
-      metadata: { requires: { topology: ['sharded'], mongodb: '>4.0.0' } },
+      metadata: { requires: { topology: ['sharded'], mongodb: '4.0.x' } },
       test: function(done) {
         const configuration = this.configuration;
-        const client = configuration.newClient(configuration.writeConcernMax(), { poolSize: 1 });
+        const client = configuration.newClient(configuration.url());
```

Comment:
Do we not need to worry about `writeConcern` here?

Reply:
In this specific case no, b/c the test should immediately error.